### DISCUSSION TITLE
No repository found for plugin `xcparse`

### DISCUSSION THREAD

**Discussion by stherold** [jdx/mise#1422] [Q&A] [answered]
I want to migrate fro [mint](https://github.com/yonaskolb/Mint) to mise to avoid having too many dependency managers. Now regarding the tool [xcparse](https://github.com/ChargePoint/xcparse), it cannot be found by mise. Is there a way to install tools from plain GH repositories?

Sorry I just discovered mise yesterday and like it a lot 🫶

**Comment by stherold** [comment]
Is it be possible to do that like below or must the plugin for xcparse exist?

`<project>/.config/mise.toml`

```bash 
[tools]
xcparse = '2.3.1'

[plugins]
xcparse = 'https://github.com/ChargePoint/xcparse'
```

**Comment by Marukome0743** [comment, marked as answer]
The URL in the proposed `[plugins]` entry points to the xcparse application repository, not to an asdf/mise plugin repository, so it cannot be used as a plugin directly.

Current mise can install xcparse directly through its GitHub backend:

```sh
mise use github:ChargePoint/xcparse@2.3.2
```

It can also be declared without creating a plugin:

```toml
[tools]
"github:ChargePoint/xcparse" = "2.3.2"
```

I verified this against the current main branch:

- `mise ls-remote github:ChargePoint/xcparse` returned the published versions through 2.3.2.
- mise downloaded and extracted the 2.3.2 release artifact.
- Running `xcparse version` through `mise exec` reported `2.3.2`.

The GitHub backend automatically selects a compatible release asset. If a different repository publishes several ambiguous binaries, its available matching and asset-selection options can be configured in the tool entry.

https://mise.en.dev/dev-tools/backends/github.html

*AI-assisted — Tool: Codex; model: OpenAI/GPT-5; version: unavailable.*
